pyproject.toml

```diff
@@ -1,7 +1,35 @@
+[build-system]
+requires = [
+    "setuptools >= 56.0",
+    "wheel >= 0.29.0",
+    "versioningit ~= 1.1.1",
+]
+build-backend = 'setuptools.build_meta'
+
+[tool.pytest.ini_options]
+junit_family = "legacy"
+addopts = "--strict-markers"
```

Comment:
Why are these pytest options here?

Reply:
no good reason, I've updated to use default settings described [here](https://docs.pytest.org/en/7.1.x/reference/customize.html)

Reply:
These are not the defaults thou but it's probably fine. 
`-q` is for quiet and makes is les verbose than default . 
`-ra` sets the summary output to all where the default is error and failure 

https://docs.pytest.org/en/6.2.x/usage.html#detailed-summary-report 

Unless there is a good reason to set something I would suggest simply deleting this section